Packs/ipinfo/Integrations/ipinfo_v2/ipinfo_v2.py

```diff
@@ -0,0 +1,197 @@
+from CommonServerPython import *  # noqa # pylint: disable=unused-wildcard-import
+from CommonServerUserPython import *  # noqa
+
+import requests
+import traceback
+from typing import Dict, Any
+
+# Disable insecure warnings
+requests.packages.urllib3.disable_warnings()  # pylint: disable=no-member
+
+
+class Client(BaseClient):
+    def __init__(self, api_key: str, base_url: str, verify_certificate: bool, proxy: bool,
+                 reliability: DBotScoreReliability):
+        """
+        Client to use in the IPinfo integration. Uses BaseClient
+        """
+        super().__init__(base_url=base_url, proxy=proxy, verify=verify_certificate)
+        self.api_key = api_key
+        self.reliability = reliability
+
+    def _url_suffix(self, ip: str) -> str:
+        api_key_in_url = f'?token={self.api_key}' if self.api_key else ''
```

Comment:
The right way to pass the `token` is via the `params` argument in `_http_request` method.
You can either provide it in `ipinfo_ip` method or create a wrapper `http_request` method, which will be called by other class methods, and will handle the `api_key`.